docs/getting-started.md

```diff
@@ -2,7 +2,101 @@
 
 This guide is here to help you get started with this repository, primarily if you've been contributing to the previous version of ownCloud's documentation, which used reStructuredText and Sphinx-Doc.
 
-## Initial Steps
+## For Non-Developers
+
+Even if you’re not a developer, you can contribute to the ownCloud documentation by [creating new issues](#creating-new-issues) and contributing to the discussion around them.
+You don’t need to be familiar with how to write or edit code, nor install any software.
+All that you need is a web browser, such as [Google Chrome](https://www.google.com/chrome/index.html), [Mozilla Firefox](https://www.mozilla.org/en-US/firefox/new/), [Apple Safari](https://www.apple.com/safari/), or [Microsoft Edge](https://www.microsoft.com/en-us/windows/microsoft-edge).
+
+### Initial Steps
+
+Before you can contribute to the documentation, you need to:
+
+1. [Have a GitHub account](#have-a-github-account).
+2. [Know the AsciiDoc basics.](#asciidoc-basics)
+
+<a name="have-a-github-account"></a>
```

Comment:
Why are you adding these links? GitHub automatically links all headlines.

Reply:
I wasn't aware that it did. I didn't see anything in their guide.

Reply:
On quick review, I see/remember that you're correct. I'll make a subsequent PR.